src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -1456,9 +1456,9 @@ public static void Remove_Invalid()
             Assert.Throws<ArgumentOutOfRangeException>("startIndex", () => builder.Remove(-1, 0)); // Start index < 0
             Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(0, -1)); // Length < 0
 
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(6, 0)); // Start index + length > 0
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(5, 1)); // Start index + length > 0
-            Assert.Throws<ArgumentOutOfRangeException>("index", () => builder.Remove(4, 2)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(6, 0)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(5, 1)); // Start index + length > 0
+            Assert.Throws<ArgumentOutOfRangeException>("length", () => builder.Remove(4, 2)); // Start index + length > 0
```

Comment:
It looks like these values changed in https://github.com/dotnet/coreclr/commit/a212082aa4cea8529b4c78844e2a7408ff6f7969.  @jkotas, @AlexGhiondea, @Clockwork-Muse, was this intentional?  If so, we'll need to specialize the test for coreclr vs desktop (or remove the name validation), as this change will cause the test to fail on desktop.